src/c++/vdo/tests/testBlkdev.c

```diff
@@ -0,0 +1,28 @@
+/*
+ * %COPYRIGHT%
+ *
+ * %LICENSE%
+ *
+ * $Id$
+ */
+
+#include <linux/bio.h>
+#include <linux/blkdev.h>
+#include <linux/blk_types.h>
+#include <linux/highmem.h>
+#include <linux/kernel.h>
+
+#include "types.h"
+
+#include "ioRequest.h"
+#include "vdoAsserts.h"
+#include "vdoTestBase.h"
+
+int blkdev_issue_zeroout(__attribute__((unused)) struct block_device *bdev,
+			 __attribute__((unused)) sector_t sector,
+			 __attribute__((unused)) sector_t nr_sects,
+			 __attribute__((unused)) gfp_t gfp_mask,
+			 __attribute__((unused)) unsigned flags)
+{
+  return VDO_SUCCESS;
```

Comment:
As mentioned, please put this implmentation in UDS. VDO uni tests can see anything in UDS user space.

But also, write an actual implmentation. This function doesn't zero anything, which means tests that use it aren't testing anything.

Reply:
I tried to point out that it didn't really need a full implementation since alloc would zero the entire device anyways. But I'll try to add one.

Reply:
I'll move the file to uds space then. I just figured it works here since vdo unit tests are only people accessing this.

Reply:
done